_docs_integrate/error-codes.md

```diff
@@ -24,74 +24,159 @@ required_by:
-| {% include anchor a="error.transport.signatureNotValid" %}                                                  | A signature is invalid.                                                                                                                                                                                                                                                   |
-| {% include anchor a="error.transport.tokens.invalidTokenContent" %}                                         | The given token content is invalid.                                                                                                                                                                                                                                       |
+# Error codes regarding the Connector <a href="https://github.com/nmshd/connector"><i class="fab fa-fw fa-github"/></a>
+
+| ErrorCode                                                                                       | Description                                                                                                                                                |
+| ----------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| {% include anchor a="error.connector.http.methodNotAllowed" %}                                  | This method is not supported for the requested resource.                                                                                                   |
+| {% include anchor a="error.connector.http.notAcceptable" %}                                     | The requested resource is capable of generating only content not acceptable according to the Accept headers sent in the request.                           |
+| {% include anchor a="error.connector.http.routeDoesNotExist" %}                                 | The requested route does not exist.                                                                                                                        |
+| {% include anchor a="error.connector.unauthorized" %}                                           | The Connector is unauthorized, because the [API-key]({% link _docs_integrate/access-the-connector.md %}) is not set or false.                              |
+| {% include anchor a="error.connector.validation.invalidJsonInPayload" %}                        | The given payload is not a valid json object.                                                                                                              |
+| {% include anchor a="error.connector.attributes.cannotCreateNotSelfOwnedRepositoryAttribute" %} | You can only create [RepositoryAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) that are owned by yourself.                    |
```

Comment:
you can find it here: https://github.com/nmshd/connector/blob/main/src/modules/coreHttpApi/controllers/AttributesController.ts
But I cannot reproduce this error because if I want create an attribute with an owner who is not me I receive "error.connector.unexpected" because it's an 500 Internal Server Error..
So I don't know if we really need this. @slavistan

Reply:
I think it is removed in release/v5 of the Connector (https://github.com/nmshd/connector/blob/release/v5/src/modules/coreHttpApi/controllers/AttributesController.ts) 😄